Scripts/Run-SS-On-Exp&Sim/scripts/fractal_dimension.py

```diff
@@ -0,0 +1,68 @@
+from scipy.spatial import ConvexHull
+import imageio
+import numpy as np
+import matplotlib.pyplot as plt
+from FractalDimension.MBdimension import fractal_dimension
+import cell_data
+
+
+def save_fig(cells, hull, points, fig_export_path, fig_name):
+    """
+    @param cells cellStates dict
+    @param hull object convex hull
+    @param points numpy array bacteria end points
+    @param  fig_export_path directory to export images
+    @param fig_name image name
+
+    """
+    # vertexes
+    vertex1_x, vertex1_y, vertex2_x, vertex2_y = cell_data.find_vertex(cells)
+    minor = cell_data.find_bacteria_minor(cells)
+
+    fig, ax = plt.subplots()
+    for i in range(len(minor)):
+        plt.plot([vertex1_x[i], vertex2_x[i]], [vertex1_y[i], vertex2_y[i]], lw=minor[i], solid_capstyle="round")
+    for simplex in hull.simplices:
+        plt.plot(points[simplex, 0], points[simplex, 1], 'k-')
+
+    fig.savefig(fig_export_path + fig_name + ".png")
+    # close fig
+    fig.clf()
+    plt.close()
+
+    # fills area
+    fig, ax = plt.subplots()
+    plt.fill(points[hull.vertices, 0], points[hull.vertices, 1], 'k')
+    fig.savefig(fig_export_path + fig_name + "_fill.png")
+    # close fig
+    fig.clf()
+    plt.close()
+
+
+def create_convex_hull(cells):
+    """
+    goal: find convex hull
+    @param cells cellStates dict
+    """
+    # find endpoints
+    endpoints_x, endpoints_y = cell_data.bacteria_end_points(cells)
+
+    points = np.column_stack((endpoints_x, endpoints_y))
+    hull = ConvexHull(points)
+
+    return hull, points
+
+
+def calc_fractal_dimension(cells, fig_export_path, fig_name):
```

Comment:
Should we calculate fractal dimension from the cell end-points or from from the colony border taken from an image? Worth discussing with @bpingalls.

Reply:
Seems the image will be a more faithful representation

Reply:
@bpingalls Done!